### ISSUE TITLE
Avoid flex-direction *-reverse values in native files

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#63092]
### Description

Splitting this out from https://github.com/WordPress/gutenberg/pull/63081

https://github.com/WordPress/gutenberg/pull/63081 adds a new Stylelint custom rule to prevent the usage of the `flex-direction` CSS property values `row-reverse` and `column-reverse` values.

For more details about why this is a problem for accessibility and the very limited cases where these values can be used, please see https://github.com/WordPress/gutenberg/issues/63048 and https://github.com/WordPress/gutenberg/issues/61241

Contributors working on the `native` files may want to consider to refactor a few components where `row-reverse` and `column-reverse` are currently in use:

- 1 occurrence in `packages/block-editor/src/hooks/use-editor-wrapper-styles.native.scss`
- 2 occurrences in `packages/block-library/src/media-text/style.native.scss`

For mow, https://github.com/WordPress/gutenberg/pull/63081 adds Stylelint disable comments. However, if the current implementation actually alters the order in a way that impacts meaning and interaction, it should be refactored.

Cc @WordPress/native-mobile 

### Step-by-step reproduction instructions

N/A

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
